### PR TITLE
Add accessibility check page

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,9 +25,9 @@ functions-build:
 check-headers-file:
 	scripts/check-headers-file.sh
 
-production-build: build check-headers-file ## Build the production site and ensure that noindex headers aren't added
+production-build: accessibility-check build check-headers-file ## Build the production site and ensure that noindex headers aren't added
```
Comment (kbhawkey) [inline Makefile]: What is accessibility-check vs. accessibility-page?

Comment (lucperkins) [reply Makefile]: This is just a WIP at the moment. I'll let you know when it's ready for review!

Comment (kbhawkey) [reply Makefile]: 👍

Comment (lucperkins) [reply Makefile]: Love the enthusiasm though 😄

**Comment by xiangpengzhao** [comment]
What is the file `yarn.lock` used for?

/cc @bep @calavera

**Comment by mistyhacks** [comment]
I'm having a hard time understanding the status of this PR. Is it ready to go?

**Comment by lucperkins** [comment]
@mistyhacks Yes, it's ready to go.

**Comment by lucperkins** [comment]
I'm honestly baffled by these CI errors. I'm gonna do some digging.

**Comment by pontiyaraja** [comment]
@kubernetes/k8s-infra-team

**Comment by lucperkins** [comment]
Initially this worked fine but something seems to have changed in one of the JS dependencies. I'll poke around and see if there's anything that can be done.

**Comment by BenTheElder** [comment]
@pontiyaraja, k8s-infra-team does not run this :-) a maintainer from the repo will know how the travis testing + netlify deploy is setup.

**Comment by pontiyaraja** [comment]
@BenTheElder thank you. 

@kubernetes/repo-infra-maintainers

**Comment by BenTheElder** [comment]
repo-infra is also a different group (https://github.com/kubernetes/repo-infra) 🙃 

@jimangel is already on this PR and should know I think

**Comment by jimangel** [comment]
> @jimangel is already on this PR and should know I think

Unfortunately this one has me scratching my head too. I will need to defer to the experts: @chenopis @zacharysarah @Bradamant3 

In the meantime I will keep my eye on this PR and do some testing too.

**Comment by lucperkins** [comment]
@jimangel I think the package’s import chain is simply busted. Nothing we can do AFAICT. I’m going to explore other avenues for accessibility checking. I’ll keep this PR open, though.

**Comment by mistyhacks** [comment]
Any update?

**Comment by lucperkins** [comment]
@mistyhacks @jimangel As far as I can tell, the import chain on this npm package is simply broken and I don't know how to address it. I'm going to close this and go back to the drawing board in terms of accessibility reporting tools.

**Comment by pontiyaraja** [comment]
@lucperkins it can be fixed if the grunt is installed in the machine where this job is getting execute. Infrastructure team should fix it.

**Comment by lucperkins** [comment]
@pointiyaraja The infrastructure team has no involvement with the website/docs, as SIG-Docs has full control over the Netlify CI environment.

As for this package, I believe that if a package can’t be installed via a plain npm installation process then the package is simply broken.

**Comment by pontiyaraja** [comment]
@lucperkins I tried by installing it from docker file and the build file, but it's not working, give me some time to fix it.

**Comment by lucperkins** [comment]
Don't worry about it. I’m going to find a different tool.

**Comment by raelga** [comment]
We are planning to start working on improving the kubernetes.io accessibility in the new git.k8s.io/community/sig-usability SIG if anyone is interested.

**Pull request closed without merging**